src/scripts/create_dbrd.py

```diff
@@ -1,59 +1,57 @@
-"""Create the DutchSocial-mini sentiment dataset and upload it to the HF Hub."""
+"""Create the DBRD-mini SENT dataset and upload it to the HF Hub."""
 
 import pandas as pd
 from datasets import Dataset, DatasetDict, Split, load_dataset
 from huggingface_hub import HfApi
 from requests import HTTPError
+from sklearn.model_selection import train_test_split
 
 
-def main() -> None:
-    """Create the DutchSocial-mini sentiment dataset and upload it to the HF Hub."""
+def main():
+    """Create the DBRD-mini SENT dataset and uploads it to the HF Hub."""
 
-    # Define the base download URL
-    repo_id = "dutch_social"
+    # Define dataset ID
+    repo_id = "dbrd"
 
     # Download the dataset
     dataset = load_dataset(path=repo_id, token=True)
     assert isinstance(dataset, DatasetDict)
 
     # Convert the dataset to a dataframe
     train_df = dataset["train"].to_pandas()
-    val_df = dataset["validation"].to_pandas()
     test_df = dataset["test"].to_pandas()
     assert isinstance(train_df, pd.DataFrame)
-    assert isinstance(val_df, pd.DataFrame)
     assert isinstance(test_df, pd.DataFrame)
 
-    # Drop all columns except for `full_text` and `label`
-    columns_to_drop = [
-        col for col in train_df.columns if col not in ["full_text", "label"]
-    ]
-    train_df.drop(columns=columns_to_drop, inplace=True)
-    val_df.drop(columns=columns_to_drop, inplace=True)
-    test_df.drop(columns=columns_to_drop, inplace=True)
+    # Strip trailing whitespace
+    train_df.text = train_df.text.str.strip()
+    test_df.text = test_df.text.str.strip()
 
-    # Rename `full_text` to `text`
-    train_df.rename(columns={"full_text": "text"}, inplace=True)
-    val_df.rename(columns={"full_text": "text"}, inplace=True)
-    test_df.rename(columns={"full_text": "text"}, inplace=True)
+    # Capitalize the first letter of each sentence
+    train_df.text = train_df.text.str.capitalize()
+    test_df.text = test_df.text.str.capitalize()
+
+    # Remove duplicates
+    train_df = train_df.drop_duplicates().reset_index(drop=True)
+    test_df = test_df.drop_duplicates().reset_index(drop=True)
 
     # Create the label column
-    label_mapping = {0: "negative", 1: "neutral", 2: "positive"}
+    label_mapping = {0: "negative", 1: "positive"}
     train_df["label"] = train_df["label"].map(label_mapping)
-    val_df["label"] = val_df["label"].map(label_mapping)
     test_df["label"] = test_df["label"].map(label_mapping)
 
-    # Create validation split
-    val_size = 256
-    val_df = val_df.sample(n=val_size, random_state=4242)
+    # Create new splits
+    _, train_df = train_test_split(
+        train_df, test_size=2048, random_state=703, stratify=train_df.label
+    )
 
-    # Create test split
-    test_size = 1024
-    test_df = test_df.sample(n=test_size, random_state=4242)
+    _, test_df = train_test_split(
+        test_df, test_size=1024, random_state=703, stratify=test_df.label
+    )
```

Comment:
The other datasets have been split using a (1024, 256, 2048) split:

```suggestion
    _, test_df = train_test_split(
        test_df, test_size=2048, random_state=703, stratify=test_df.label
    )
```